src/dbnode/persist/fs/files.go

```diff
@@ -556,9 +642,16 @@ func forEachInfoFile(
 		case persist.FileSetFlushType:
 			switch args.contentType {
 			case persist.FileSetDataContentType:
-				checkpointFilePath = filesetPathFromTime(dir, t, checkpointFileSuffix)
-				digestsFilePath = filesetPathFromTime(dir, t, digestFileSuffix)
-				infoFilePath = filesetPathFromTime(dir, t, infoFileSuffix)
+				isLegacy := false
```

Comment:
We should perhaps leave some comments somewhere in this package how we could eventually remove these legacy pieces of logic if we ever cut a 1.0 release and can perform backwards incompatible changes.

Just like a few sentences on the thinking about how to remove it and which string to "grep" for in the code to find all the places we can delete dead code once we don't need to support backwards compat for it.

Reply:
👍